srcpkgs/beancount/template

```diff
@@ -1,16 +1,25 @@
 # Template file for 'beancount'
 pkgname=beancount
-version=2.3.0
-revision=3
+version=2.3.5
+revision=1
 build_style=python3-module
+make_check_args="--pyargs beancount --ignore beancount/web/web_test.py
+ --deselect beancount/scripts/bake_test.py::TestScriptBake::test_bake_directory
+ --deselect beancount/ingest/extract_test.py::TestScriptExtract::test_extract_no_files"
 hostmakedepends="python3-setuptools"
 makedepends="python3-devel"
 depends="python3-pytest python3-dateutil python3-ply python3-bottle python3-lxml
```

Comment:
does pytest need to be in depends?

Reply:
Yes that seemed weird to me but they still have it under the [`install_requires`](https://github.com/beancount/beancount/blob/2.3.5/setup.py#L80-L82) in this version so I didn't touch it.

Reply:
i think that's just them not knowing how to use install_requires, unless you can find them using pytest somewhere

Reply:
It seems to be used in [this module](https://github.com/beancount/beancount/blob/2.3.5/beancount/ingest/regression_pytest.py). If you see the master branch [`setup.py`](https://github.com/beancount/beancount/blob/master/setup.py#L80-L96) pytest got removed because now there is no [ingest module](https://github.com/beancount/beancount/tree/master/beancount/ingest)! So I'm assuming that's related.

Reply:
hm this seems like something that shouldn't be installed but idk

Reply:
It's fine to wait for the next release from upstream then. It appears to clean up some of the deps.